data/hai missions.txt

```diff
@@ -435,3 +435,566 @@ mission "Unfettered returning home"
+		conversation
+			`Turner meets you in a high-end human-owned restaurant in the richer part of the city, very much reminiscent of something you would find on a Paradise Planet. "It's been quite some time since we last spoke, Captain <last>. Over a year now," he says. "Have you been enjoying the extra cash in your pocket every day, Captain <last>?"`
+			choice
+				`	"Yes. It's been a huge help paying crew salaries."`
+					goto salaries
+				`	"Five thousand credits each day isn't much for me."`
+			
+			`	"I suppose just under two million credits every year wouldn't be much to a hero such as yourself. Spaceships are expensive, after all. How does five million credits every year sound to you?"`
+				goto choice
+			
+			label salaries
+			`	"I'm glad to hear that, Captain <last>. I spent my early years as a captain myself, and I didn't much enjoy having to worry about crew salaries every day. Perhaps nearly tripling that amount will help even more."`
+			
+			label choice
+			choice
+				`	"Am I getting a raise?"`
+				`	"Is this part of your new proposition?"`
+					goto proposition
+			
+			`	"You could call it that," Turner says after sipping from his glass of wine.`
+			
+			label proposition
+			`	"Building a outfitter selling human goods here on <origin> has had to be the best idea I've ever had, Captain <last>, but recently I thought of an even better idea. You see, most of the humans you see flying around in Hai space are flying decades-old ships. Hai ships are prohibitively expensive for humans. The Aphid is nearly three million credits! This means that demand for ships is extremely high among humans, Captain <last>. Do you see where I'm going with this?"`
+			choice
+				`	"You need me to help you escort freighters again?"`
+				`	"Do you want to build a shipyard selling human ships?"`
+			
+			`	"Yes, Captain. Right now, the only realistic way that a human born here can get a ship is if they know a captain willing to retire. Even then, prices for a human ship can be more than twice what they are across the wormhole. It's preposterous, Captain <last>!`
+			`	"I've already made arrangements with the local Hai government. The rest of the park next to the outfitter is now my property, and I intend to use it to build the shipyard. I've also contacted Megaparsec and bought a license to build their ships and some of the outfits they sell on <planet>. My freighters are at the ready in the spaceport right now. All I need you to do is escort them to <destination> and back, and in six to seven months' time when the shipyard is done, you'll reap the rewards. What do you say, Captain <last>? Do you want to be a hero to millions once again?"`
+			choice
+				`	"Sure, this sounds like a great idea."`
+					goto accept
+				`	"I'm not interested in working with you anymore, Turner."`
+			
+			`	Turner smirks at your response. "Understood, Captain <last>. I'll just save myself five thousand credits a day from now on. I'll be seeing myself out now."`
```

Comment:
Just dropping the salary like that is a bit mean, I think we need more moments like that but others may not be so happy with it. Adding in a line where he warns the player that's what will happen if they don't do what he wants will resolve this.